README.md

```diff
@@ -1 +1,46 @@
-# export-service-go
\ No newline at end of file
+# export-service-go
+
+## Dependencies
+- Golang >= 1.16
+- podman-compose
+- make (optional for running the Makefile)
+## Starting the service
+You can start the database and minio using `podman-compose up db s3`. This will expose minio on `localhost:9099` 
+(use `minio` as the access key and `minioadmin` as the secret key)
+
+Create a bucket called `exports-bucket` in minio, click **Manage** and set it's **Access Policy** to `public`.
```

Comment:
Just a note on this part, it's strange to do this manually, so I made a ticket to try and automate it. [RHCLOUD-21616](https://issues.redhat.com/browse/RHCLOUD-21616)